index.js

```diff
@@ -100,16 +100,28 @@ function getOrder(a, b) {
 	return 0;
 }
 
-function ExtractTextPlugin(id, filename, options) {
-	if(typeof filename !== "string") {
-		options = filename;
-		filename = id;
-		id = ++nextId;
+function ExtractTextPlugin(options) {
+	if(arguments.length > 1) {
+		throw new Error("Deprecation notice: ExtractTextPlugin now only takes a single argument. Either an options " +
```

Comment:
`Deprecation`s normally mean that: "This still works the way you're using it, but it'll stop working soon." Whereas this PR causes the old API to not work at all. I recommend either 1) we make it support both APIs, 2) change the words `Deprecation notice` to `Breaking change`